src/main/java/step1/domain/Numbers.java

```diff
@@ -0,0 +1,25 @@
+package step1.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Numbers {
+
+    private static Pattern pattern = Pattern.compile("\\d+");
```

Comment:
```suggestion
    private static final Pattern pattern = Pattern.compile("\\d+");
```
변경되지 않도록 final 키워드를 붙여주면 좋습니다.